bundles/org.openhab.binding.iaqualink/src/main/java/org/openhab/binding/iaqualink/internal/api/model/AccountInfo.java

```diff
@@ -0,0 +1,236 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.iaqualink.internal.api.model;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Account Info Object
+ *
+ * @author Dan Cunningham - Initial contribution
+ *
+ */
+public class AccountInfo {
+
+    @SerializedName("id")
+    @Expose
```

Comment:
Isn't expose the default

Reply:
This was machine generated from the source json files, i can remove these if they are not to you liking, but imho being verbose with annotations is not really hurting anything.

Reply:
FYI I did remove these